pkg/components/components.go

```diff
@@ -1242,7 +1242,7 @@ func InjectVolumesForWebHookCerts(deploy *appsv1.Deployment) {
 		deploy.Spec.Template.Spec.Containers[index].VolumeMounts = append(container.VolumeMounts,
 			corev1.VolumeMount{
 				Name:      "apiservice-cert",
-				MountPath: hcov1beta1.WebhookCertDir,
+				MountPath: hcov1beta1.GetWebhookCertDir(),
```

Comment:
I think this one is not needed.  Here we're creating the mount for the container, so we need the constant here.

Reply:
fixed.